docs/release-notes/8.1.asciidoc

```diff
@@ -1,6 +1,18 @@
 [[release-notes-header-8.1.0]]
 == 8.1
 
+[discrete]
+[[release-notes-8.1.1]]
+=== 8.1.1
+
+[discrete]
+[[bug-fixes-8.1.1]]
+==== Bug fixes and enhancements
+* Fixes a bug in the Edit Rule Exception flyout that was truncating error messages ({pull}126875[#126875]).
```

Comment:
Can we hold off on this one? It looks like it was only partially fixed by my PR. Info in this issue here - https://github.com/elastic/kibana/issues/126669

Reply:
Yeah, of course! To clarify, did you want me to remove this entirely or wait until later in the 8.1.1 cycle to release note it? I can also modify the summary if you'd prefer to take that route.

Reply:
Verified through Slack to remove this.